### ISSUE TITLE
libboost error running ./GuldenD

### ISSUE THREAD

**Issue by drothroc** [Gulden/gulden-official#64]
Hi, trying to setup and I run into the error below:
`./GuldenD: error while loading shared libraries: libboost_system.so.1.61.0: cannot open shared object file: No such file or directory
`
When I run locate here is the output, it seems when I followed the directions for ubuntu 14, it installed boost 1.61 into the gulden directory, however, my usr/lib is version 1.54.0.

`~/gulden-official/src$ locate libboost_system
/boost166/lib/libboost_system.a
/boost166/lib/libboost_system.so
/boost166/lib/libboost_system.so.1.61.0
/home/test123/gulden-official/boost_1_61_0/bin.v2/libs/system/build/gcc-4.8/debu                                                                                      g/threading-multi/libboost_system.so.1.61.0
`
`
/home/test123/gulden-official/boost_1_61_0/bin.v2/libs/system/build/gcc-4.8/rele                                                                                      ase/link-static/threading-multi/libboost_system.a
`
`
/home/test123/gulden-official/boost_1_61_0/bin.v2/libs/system/build/gcc-4.8/rele                                                                                      ase/threading-multi/libboost_system.so.1.61.0
`
`
/usr/lib/x86_64-linux-gnu/libboost_system.a
/usr/lib/x86_64-linux-gnu/libboost_system.so
/usr/lib/x86_64-linux-gnu/libboost_system.so.1.54.0
`
Any help would be appreciated, I think I either need to change the directory GuldenD is looking for or update my usr/libs.

**Comment by mjmacleod** [comment]
If you compiled with boost in a non-system path then you need to run with a custom LD path.
e.g. `LD_LIBRARY_PATH="/boost160/" ./GuldenD` or similar.

**Comment by mjmacleod** [comment]
Or alternatively force your boost to build only as 'static' so that GuldenD static links to it and doesn't have to search at runtime.

IIRC the instructions here https://gist.github.com/mjmacleod/a3562af661661ce6206e5950e406ff9d do this (force static only boost)

**Comment by drothroc** [comment]
Thank you for the quick response, i've ended up rebuilding, which took some time, but got the same error... When I launched the daemon with LD_LIBRARY_PATH="/boost166/lib" ./GuldenD which solved the problem, however, it gets hung up. Where is the .conf file that I need to edit? rpc, ports, etc. I've only compiled and ran litecoin, which had a litecoin.conf that I had to create and insert into a folder. I'm not sure what to do with Gulden.

Thanks again for your help.
